.master-plan.yml

```diff
@@ -983,6 +983,14 @@ spec:
       test/tags: "OPENEBS, LOCALPV"
       testplan: "https://e2e.mayadata.io/docs/openebs-enterprise/OPENEBS-FUNCTIONAL/TCID-LOCALPV-SELECTED-DEVICE-CREATE"
 
+  - tcid: TCID-LOCALPV-SELECTED-DEVICE-RECONCILE
+    name: "TCID-LOCALPV-SELECTED-DEVICE-RECONCILE"
+    description: "Test creation of OpenEBS LocalPV reconiliation using selected block device"
```

Comment:
Reconciliation

Reply:
Test if localPV can be reconciled successfully after labeling a selected block device